src/Middleware/HttpsPolicy/src/HttpsRedirectionOptions.cs

```diff
@@ -6,7 +6,7 @@
 namespace Microsoft.AspNetCore.HttpsPolicy
 {
     /// <summary>
-    /// Options for the HttpsRedirection middleware
+    /// Options for the HttpsRedirection middleware.
```

Comment:
```suggestion
    /// Options for the HTTPS Redirection Middleware.
```

I'd like to avoid using the type name here (I know it was there in the original code, but since you're editing these docs...). Most users interact with this through `UseHttpsRedirection()` which means they aren't seeing the `HttpsRedirection` type.